src/main/java/io/antmedia/streamsource/StreamFetcherManager.java

```diff
@@ -89,7 +90,12 @@ public void setRestartStreamFetcherPeriod(int restartStreamFetcherPeriod) {
 
 
 	public StreamFetcher startStreaming(Broadcast broadcast) {	
-
+		if(scope.getContext().hasBean(IResourceMonitor.BEAN_NAME)) {
+			IResourceMonitor monitor = (IResourceMonitor) scope.getContext().getBean(IResourceMonitor.BEAN_NAME);
+			if(monitor.getAvgCpuUsage() > monitor.getCpuLimit()) {
+				return null;
```

Comment:
Please write a simple log warning/error at least, in order to show that stream is not accepted due to cpu limit